upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -209,16 +209,88 @@
         </div>
         <div id="tab-rss" style="display: none">
           <table class="form">
+            <tr>
+              <td>
+                <?php echo $rss_category_title; ?>
+              </td>
+              <td>
+                <select name="smaily_for_opencart_rss_category" id="rss-category" class="smaily-rss-options">
+                    <option value=''>All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = (int) $category['category_id'] === $rss_category ? 'selected' : '';
+                      echo("<option value=$category[category_id] $selected>$category[name]</option>");
+                    }
+                    ?>
+                </select>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <?php echo $rss_sort_by_title; ?>
+              </td>
+              <td>
+                <select name="smaily_for_opencart_rss_sort_by" id="rss-sort-by" class="smaily-rss-options">
+                   <?php
+                    // All available options
+                    $sort_options = [
+                      'pd.name' => $sort_name,
+                      'p.model' => $sort_model,
+                      'p.price' => $sort_price,
+                      'p.status' => $sort_status,
+                      'p.sort_order' => $sort_order
+                    ];
+                    // Add options for select.
+                    foreach ($sort_options as $sort_code => $sort_name) {
+                      $selected = $rss_sort_by === $sort_code ? 'selected' : '';
+                      echo("<option value='$sort_code' $selected>$sort_name</option>");
+                    }
```

Comment:
Use the same pattern here, as with category field.

Reply:
Not echoing out html anymore